src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js

```diff
@@ -0,0 +1,260 @@
+
+    const formatNewParams = useCallback((voltageLimits) => {
+        return {
+            [VOLTAGE_LIMITS]: voltageLimits.map((voltageLimit) => {
+                return {
+                    [PRIORITY]: voltageLimits.indexOf(voltageLimit),
+                    [LOW_VOLTAGE_LIMIT]: voltageLimit[LOW_VOLTAGE_LIMIT] ?? 0,
+                    [HIGH_VOLTAGE_LIMIT]: voltageLimit[HIGH_VOLTAGE_LIMIT] ?? 0,
+                    [FILTERS]: voltageLimit[FILTERS].map((filter) => {
+                        return {
+                            [FILTER_ID]: filter[ID],
+                            [FILTER_NAME]: filter[NAME],
+                        };
+                    }),
+                };
+            }),
+        };
+    }, []);
+
+    const onSubmit = useCallback(
+        (newParams) => {
+            updateVoltageInitParameters(
+                studyUuid,
+                formatNewParams(newParams.voltageLimits)
+            )
+                .then(() => {
+                    setVoltageInitParams(
+                        formatNewParams(newParams.voltageLimits)
+                    );
+                })
+                .catch((error) => {
+                    snackError({
+                        messageTxt: error.message,
+                        headerId: 'DeleteVoltageLevelOnLineError',
+                    });
+                });
+        },
+        [formatNewParams, setVoltageInitParams, snackError, studyUuid]
+    );
+
+    const resetVoltageInitParameters = useCallback(() => {
+        updateVoltageInitParameters(studyUuid, emptyFormData)
+            .then(() => {
+                return getVoltageInitParameters(studyUuid)
+                    .then((params) => setVoltageInitParams(params))
+                    .catch((error) => {
+                        snackError({
+                            messageTxt: error.message,
+                            headerId: 'paramsRetrievingError',
+                        });
+                    });
+            })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'paramsChangingError',
+                });
+            });
+    }, [studyUuid, emptyFormData, setVoltageInitParams, snackError]);
+
+    const clear = useCallback(() => {
+        reset(emptyFormData);
+        resetVoltageInitParameters();
+    }, [emptyFormData, reset, resetVoltageInitParameters]);
+
+    return (
+        <FormProvider validationSchema={formSchema} {...formMethods}>
+            <DndTable
+                arrayFormName={`${VOLTAGE_LIMITS}`}
+                columnsDefinition={COLUMNS_DEFINITIONS}
+                useFieldArrayOutput={useFieldArrayOutput}
+                createRows={createVoltageLimitRows}
+                tableHeight={270}
+                withAddRowsDialog={false}
+                withLeftButtons={false}
+            />
+
+            <DialogActions>
```

Comment:
You should add a close button like other parameters tabs

Reply:
agreed, it's done